spec/ddtrace/benchmark/pg_contrib_spec.rb

```diff
@@ -0,0 +1,263 @@
+# typed: ignore
+
+require 'spec_helper'
+
+require 'benchmark'
+require 'ddtrace'
+require 'pg'
+
+RSpec.describe 'Pg Tracing Integration', :order => :defined do
```

Comment:
Because we currently don't have a way to capture benchmark results in CI, we'd be running it and pretty much not inspecting the results. This means we'd be making CI slower with very little value.

I recommend adding a conditional that makes it not run in CI at the moment:
```ruby
before { skip('Benchmark results not currently captured in CI') if ENV.key?('CI') }
```